app.py

```diff
@@ -3,106 +3,243 @@
 import os
 from flask import Flask, jsonify, make_response, Response, request
 
+from paper_trader.models.user_model import find_user_by_id, update_user_balance
+from paper_trader.models.user_stock_model import (
+    find_stock_by_user_and_symbol,
+    update_stock_quantity,
+    add_new_stock,
+    remove_stock,
+)
+
 from utils.stocks import quote_stock_by_symbol
 
 # Load environment variables
 load_dotenv()
 
 app = Flask(__name__)
 
+
 # Health Checks
-@app.route('/health', methods=['GET'])
+@app.route("/health", methods=["GET"])
 def healthcheck():
-    '''
+    """
     Health Check to ensure the service is running and healthy
-    
+
     Returns:
         JSON response indicating the status of the service
-    '''
-    res = {
-        'status': 'ok'
-        }
-    
-    app.logger.info('Health Check')
+    """
+    res = {"status": "ok"}
+
+    app.logger.info("Health Check")
     return make_response(jsonify(res), http.HTTPStatus.OK)
 
+
 # Authentication
-@app.route('/auth/login', methods=['POST'])
+@app.route("/auth/login", methods=["POST"])
 def login():
-    '''
+    """
     Login endpoint to authenticate the user
-    
+
     Returns:
         JSON response indicating the status of the login
-    '''
+    """
     pass
 
-@app.route('/auth/create-account', methods=['POST'])
+
+@app.route("/auth/create-account", methods=["POST"])
 def register():
-    '''
+    """
     Register endpoint to create a new user
-    
+
     Returns:
         JSON response indicating the status of the registration
-    '''
+    """
     pass
 
-@app.route('/auth/change-password', methods=['PATCH'])
+
+@app.route("/auth/change-password", methods=["PATCH"])
 
 # Stock Management
-@app.route('/stocks/buy', methods=['POST'])
+@app.route("/stocks/buy", methods=["POST"])
 def buy_stock():
```

Comment:
logic looks good but can we move everything except for the responses into `user_stock_model`?
ex: we should have a function `user_stock_model.buy_stock(user_id: str, symbol: str, quantity: float)` so we can make the tests correctly for it